Makefile

```diff
@@ -21,9 +21,7 @@ test-integration:
 
 build-production: test
```

Comment:
Is it worth having this anymore, now we use goreleaser? It might be better to drop the build-production target and just rely on build, then have goreleaser do the build matrix.

Reply:
I find it quite useful to test stuff during development - I can just `make build-production` to get a fresh binary that I can test out. I don't know if goreleaser supports building without releasing to everywhere, but that's what I'd like to support.

Reply:
Actually what I've said doesn't make much sense - now we have one binary, `make build` is sufficient for local testing. The only place where we still use this is in CircleCI, to generate the binary for integration tests.